app/views/layouts/application.html.erb

```diff
@@ -14,11 +14,26 @@
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
 
-      <% if current_user %>
+      <% if current_admin? %>
+        <%= link_to "Users", "/admin/users" %>
+        <%= link_to "Dashboard", "/admin" %>
+        <%= link_to "Profile", "/profile" %>
+        <%= link_to "Logout", "/logout", method: :delete %>
+        <div>Logged in as <%= current_user.name %></div>
+
+      <% elsif current_merchant? %>
+        <%= link_to "Dashboard", "/merchant" %>
+        <%= link_to "Cart: #{cart.total_items}", "/cart" %>
+        <%= link_to "Profile", "/profile" %>
+        <%= link_to "Logout", "/logout", method: :delete %>
+        <div>Logged in as <%= current_user.name %></div>
+
+      <% elsif current_user %>
         <%= link_to "Cart: #{cart.total_items}", "/cart" %>
         <%= link_to "Profile", "/profile" %>
         <%= link_to "Logout", "/logout", method: :delete %>
         <div>Logged in as <%= current_user.name %></div>
+
```

Comment:
Looks good. We could refactor this later using partials to bring the overall size of the code down.